Adafruit_MotorHAT/Adafruit_MotorHAT_Motors.py

```diff
@@ -256,7 +255,7 @@ def setPin(self, pin, value):
     def getStepper(self, steps, num):
         if (num < 1) or (num > 2):
             raise NameError('MotorHAT Stepper must be between 1 and 2 inclusive')
-        return self.steppers[num-1]
+        return Adafruit_StepperMotor(self, num, steps)
```

Comment:
Doesn't this make a new stepper, instead of returning one already created in __init__?